tools/spy-detector/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "spy-detector"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+nng = "1.0.1"
```

Comment:
no need to have `nng` as a dependency here. You can use it via `shared`.

Drop this 
```suggestion
```
and use it similar to

https://github.com/0xB10C/peer-observer/blob/8a445f71608bfd968c1231fbfa5b5864f06ab53c/tools/websocket/src/main.rs#L11-L12

Reply:
Okay, done!